### DISCUSSION TITLE
KOReader and Kobo FW

### DISCUSSION THREAD

**Discussion by mergen3107** [koreader/koreader#9149] [Q&A]
I recently installed KOReader on Kobo Clara HD with FW **4.22.15190**.

Is there a solid reason to update FW version? Does KOReader benefit from newer FW?
Or, in the contrary, are there something that breaks on newer FW?

Thanks!

**Comment by Frenzie** [comment]
The first few months up to let's say a year after the device came out, maybe. After that I rather doubt it, generally speaking.

**Reply by NiLuJe** [reply]
Kernel updates might matter. I don't own a Clara, so I can't recall when it was last updated, but, IIRC, it was updated for the Forma in 4.23.

TL;DR: There's no solid reason *NOT* to update, if you can spare 10 minutes ;).

**Comment by mergen3107** [comment]
Thank you guys :)
